manifests/script-lab-edge.olk.xml

```diff
@@ -0,0 +1,330 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Actually, now that I think about it.... I think you'll need to copy the string localizations too from the other manifests.